source/GrowthbeatCore/GrowthbeatCore/GrowthbeatCore.h

```diff
@@ -35,5 +35,5 @@
 - (GBClient *)waitClient;
 
 - (BOOL)handleIntent:(GBIntent *)intent;
-
+- (void)implementIntentHandler:(void(^)(GBIntent *intent))block;
```

Comment:
Implement `addIntentHandler:` method instead of this to add multiple custom intent handlers.